### PR TITLE
#12 Multiple Boards

### PR DESCRIPTION
**Pull request by Guru834**
### What this PR does (required):
- When the user clicks on Create Board button in nabber, dialog appears.
- In that dialog user can type the Board Title.
- On submit the board title will appear in the sidebar

### Screenshots / Videos (front-end only):

Screen Shot 2021-11-08 at 10 35 37 PM

Screen Shot 2021-11-08 at 10 36 23 PM

Screen Shot 2021-11-08 at 10 36 51 PM

### CODE REVIEW

File: client/src/components/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -1,15 +1,69 @@
...
+  const DialogTitleIcon = (props: Props) => {
+    const { onClose, ...other } = props;
+    return (
+      <MuiDialogTitle className={classes.iconTitleBox} disableTypography {...other}>
+        <Typography variant="h5" className={classes.cardTitle}>
+          {onClose ? (
+            <IconButton aria-label="close" onClick={onClose}>
+              <Close />
+            </IconButton>
+          ) : null}
```
Comment (ezeilo-su) [inline client/src/components/Navbar/Navbar.tsx:44]: I personally don't like returning a `null` content. Instead I would put this code like this:
`
{!!onClose && (
            <IconButton aria-label="close" onClick={onClose}>
              <Close />
            </IconButton>
          ) }
`
So it renders only when `onClose` (converted to boolean with `!!`) is `true`
This is a personal preference, though :smiley:

Comment (ezeilo-su) [inline client/src/components/Navbar/Navbar.tsx:44]: I personally don't like returning a `null` content. Instead I would put this code like this:
`
{!!onClose && (
            <IconButton aria-label="close" onClick={onClose}>
              <Close />
            </IconButton>
          ) }
`
So it renders only when `onClose` (converted to boolean with `!!`) is `true`
This is a personal preference, though :smiley:

**Review by ezeilo-su — COMMENTED** [review]
LGTM!

**Review by ezeilo-su — COMMENTED** [review]
LGTM!

**Review by moffatethan — APPROVED** [review]

**Pull request merged**